src/main/resources/init_db.sql

```diff
@@ -6,3 +6,35 @@ CREATE TABLE `taxi_service`.`manufacturers` (
         `manufacturer_country` VARCHAR(225) NOT NULL,
         `deleted` TINYINT NOT NULL DEFAULT 0,
         PRIMARY KEY (`manufacturer_id`));
+CREATE TABLE `taxi_service`.`drivers` (
+        `driver_id` BIGINT(11) NOT NULL AUTO_INCREMENT,
+        `driver_name` VARCHAR(225) NOT NULL,
+        `driver_license_number` VARCHAR(225) NOT NULL,
```

Comment:
there is no need to indicate `driver_` prefix for each column because the columns are located in `drivers` table

same for other column names in manufacturer and car tables